Sources/Xcodeproj/pbxproj.swift

```diff
@@ -108,15 +108,8 @@ public func xcodeProject(
     // Set the SUPPORTED_PLATFORMS to all platforms.
     // FIXME: This doesn't seem correct, but was what the old project generation
     // code did, so for now we do so too.
-    projectSettings.common.SUPPORTED_PLATFORMS = [
-        "macosx",
-        "iphoneos",
-        "iphonesimulator",
-        "appletvos",
-        "appletvsimulator",
-        "watchos",
-        "watchsimulator",
-    ]
+    projectSettings.common.SUPPORTED_PLATFORMS = ["$(AVAILABLE_PLATFORMS)"]
```

Comment:
Is there a minimum Xcode version that defines `AVAILABLE_PLATFORMS`, or would this work on, say, 11.x?

Reply:
This setting must be at least 10 years old by now.